### PR TITLE
Add > Steampipe Cloud > Integrations Doc > DBeaver

### PR DESCRIPTION
**Pull request by RahulSrivastav14**

### CODE REVIEW

File: docs/cloud/integrations/dbeaver.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+[DBeaver](https://dbeaver.io/download/) Community is open source and free to use on the desktop, while the PRO version can be purchased. In this example we will create a Steampipe Cloud connection with DBeaver and query the AWS S3 buckets.
```
Comment (judell) [inline docs/cloud/integrations/dbeaver.md:36]:
```suggestion
[DBeaver](https://dbeaver.io/download/) Community is open source and free to use on the desktop, while the pro version can be purchased. In this example we will create a Steampipe Cloud connection with DBeaver and query the AWS S3 buckets.
```

File: docs/cloud/integrations/dbeaver.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Here we select the `aws_s3_bucket` table. DBeaver displays the table's schema and previews the returned data. You can drag the columns to re-arrange the data and also export it to a wide range of formats like CSV, JSON, SQL, TXT, XML, etc.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/dbeaver-bucket-query-result.png" />
+</div>
+
+## Run your first custom query
+
+DBeaver comes equipped with a SQL query editor that you may use to write a custom query. For example, we can use this query in the editor to fetch the list of AWS EC2 instance types available in us-east-1 region.
```
Comment (judell) [inline docs/cloud/integrations/dbeaver.md:58]:
```suggestion
DBeaver provides a SQL query editor that you may use to write a custom query. For example, we can use this query in the editor to fetch the list of AWS EC2 instance types available in us-east-1 region.
```

**Review by judell — CHANGES_REQUESTED** [review]
Looks good! Just a few tweaks, thank you.

File: docs/cloud/integrations/dbeaver.md (hunk 1 above)
Comment (RahulSrivastav14) [reply docs/cloud/integrations/dbeaver.md:36]: Rectified

File: docs/cloud/integrations/dbeaver.md (hunk 2 above)
Comment (RahulSrivastav14) [reply docs/cloud/integrations/dbeaver.md:58]: Rectified

**Pull request merged**